ttk/install_ttk.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Build TTK
+if [ ! -d ttk-source ]; then
+  git clone https://github.com/topology-tool-kit/ttk.git ttk-source
+  mkdir ttk-build
+  cd ttk-source
+  git checkout a26adce85a53a7581dfd4fc7bef96305db8a9383
+  rm -rf paraview/WRLExporter/ core/vtk/ttkWRLExporter/ 
```

Comment:
This is currently required. A fix in TTK will be merged soon: https://github.com/topology-tool-kit/ttk/pull/451

Reply:
this is merged now so we can bump the commit hash and remove these lines.

Reply:
Actually, according to @julien-tierny, it seems that the current dev branch is not stable so stick with this commit for now.